packages/core/src/controllers/pairing.ts

```diff
@@ -187,14 +187,15 @@ export class Pairing implements IPairing {
 
   public activate: IPairing["activate"] = async ({ topic }) => {
     this.isInitialized();
-    const expiry = calcExpiry(THIRTY_DAYS);
+    const expiry = calcExpiry(FIVE_MINUTES);
     this.core.expirer.set(topic, expiry);
     await this.pairings.update(topic, { active: true, expiry });
   };
 
   public ping: IPairing["ping"] = async (params) => {
```

Comment:
We need to add JSDoc so people see it in their IDEs.
```suggestion
 /**
 * @deprecated Ping will be removed in the next major release.
 */
  public ping: IPairing["ping"] = async (params) => {
```